docs/builds.rst

```diff
@@ -1,16 +1,12 @@
 Build Process
 =============
 
-Files: `tasks.py`_ - `doc_builder/`_
-
-.. _tasks.py: https://github.com/readthedocs/readthedocs.org/blob/master/readthedocs/projects/tasks.py
-.. _doc_builder/: https://github.com/rtfd/readthedocs.org/tree/master/readthedocs/doc_builder
-
 Every documentation build has limited resources.
 Our current build limits are:
 
 * 15 minutes of CPU
 * 1GB of RAM memory
```

Comment:
These changed as well, no?

Reply:
Yes. I think the default is 15m and 3GB now. I'll update.